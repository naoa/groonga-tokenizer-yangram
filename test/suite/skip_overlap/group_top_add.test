register tokenizers/yangram

tokenize TokenYaBigram "Tommorowは雨だな。" NormalizerAuto --mode ADD
