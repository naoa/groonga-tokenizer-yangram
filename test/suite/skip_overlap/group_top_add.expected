register tokenizers/yangram
[[0,0.0,0.0],true]
tokenize TokenYaBigram "Tommorowは雨だな。" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "tommorow",
      "position": 0,
      "force_prefix": false
    },
    {
      "value": "は雨",
      "position": 1,
      "force_prefix": false
    },
    {
      "value": "雨だ",
      "position": 2,
      "force_prefix": false
    },
    {
      "value": "だな",
      "position": 3,
      "force_prefix": false
    },
    {
      "value": "な",
      "position": 4,
      "force_prefix": false
    },
    {
      "value": "。",
      "position": 5,
      "force_prefix": false
    }
  ]
]
